university-of-aleppo-faculty-of-medicine.csl

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>University of Aleppo - Faculty of Medicine</title>
+    <title-short>UAFM</title-short>
+    <id>http://www.zotero.org/styles/university-of-aleppo-faculty-of-medicine</id>
+    <link rel="self" href="http://www.zotero.org/styles/university-of-aleppo-faculty-of-medicine"/>
```

Comment:
add a link with rel="documentation" to the documentation you created

Reply:
Done